code/modules/vehicles/scooter.dm

```diff
@@ -12,38 +12,34 @@
 	AddElement(/datum/element/ridable, /datum/component/riding/vehicle/scooter)
 
 /obj/vehicle/ridden/scooter/wrench_act(mob/living/user, obj/item/I)
-	..()
+	. = ..()
```

Comment:
why set . if you override the default return value all the time?

Reply:
Sure I guess